app/controllers/users_controller.rb

```diff
@@ -83,6 +83,27 @@ def resend_invitation
     redirect_to update_user_path(user_id: params[:user_id])
   end
 
+  def show_update_email
+    @form = UpdateUserEmailForm.new({})
+  end
+
+  def update_email
+    @form = UpdateUserEmailForm.new(params["update_user_email_form"])
+    if @form.valid?
+      update_user_email(user_id: params["user_id"], email: @form.email)
+      redirect_to update_user_path
+    else
+      flash.now[:errors] = @form.errors.full_messages.join(', ')
```

Comment:
we should be using the GOV.UK styles for the error messages, but it can be left for the next PR

Reply:
I've done it in https://github.com/alphagov/verify-self-service/pull/234/commits/ae5794acafce356dcb3ccb1b5ad923f04a3f876e